lib/osf-components/addon/components/validated-input/text/styles.scss

```diff
@@ -1,5 +1,14 @@
-.PrefixedInputBox {
-    display: flex !important;
+:global(.form-control).PrefixedInputBox {
+    display: flex;
+    padding: 0;
+}
+
+.Prefix {
+    background-color: $color-bg-gray-light;
+    border-right: solid 1px $color-border-gray;
+    padding: 6px 12px;
```

Comment:
Would reducing the left-right padding make it more likely we could see the border? Or is it just bad choices all around?

Reply:
Yea, it looks like lowering the side-padding does the trick! Thank you!

Reply:
Also, color contrast is 6.48, so it's AA compliant, but not AAA compliant 👀